packages/app-store/salesforce/lib/CalendarService.ts

```diff
@@ -196,7 +196,6 @@ export default class SalesforceCalendarService implements Calendar {
   }
 
   async createEvent(event: CalendarEvent): Promise<NewCalendarEventType> {
-    debugger;
```

Comment:
Removing as I searched for "debugger" in the entire app to remove mine and found this one too